test/endtoend/wdio.remote.conf.js

```diff
@@ -14,7 +14,7 @@ exports.config = Object.assign(base.config, {
     { browserName: 'chrome', version: '50.0', platform: 'Windows 7' },
     { browserName: 'MicrosoftEdge', version: '14.14393', platform: 'Windows 10' },
     // { browserName: 'safari', version: '11.0', platform: 'macOS 10.12' },
-    { browserName: 'safari', version: '11.0', platformName: 'iOS', platformVersion: '10.0', deviceOrientation: 'portrait', deviceName: 'iPhone 6 Simulator', appiumVersion: '1.7.1' },
+    { browserName: 'safari', version: '10.0', platformName: 'iOS', platformVersion: '10.0', deviceOrientation: 'portrait', deviceName: 'iPhone 6 Simulator', appiumVersion: '1.7.1' },
     { browserName: 'chrome', platformName: 'Android', platformVersion: '6.0', deviceOrientation: 'portrait', deviceName: 'Android Emulator', appiumVersion: '1.7.1' }
```

Comment:
🥇wow, thanks!

Reply:
Wait. So Safari on Mac OS is still disabled: https://github.com/transloadit/uppy/blob/master/test/endtoend/wdio.remote.conf.js#L16. Maybe we should try `10.0` there as well.

Reply:
@arturi I set 10 for safari on iOS(not macOS). It was breaking before I set it to 10. So maybe we should set it for macOS too?